admin/server/organizations.go

```diff
@@ -44,6 +44,18 @@ func (s *Server) GetOrganization(ctx context.Context, req *adminv1.GetOrganizati
 	}
 
 	if !claims.CanOrganization(ctx, org.ID, auth.ReadOrg) {
+		// check if the user is outside members of a project in the org
+		projects, err := s.admin.DB.FindProjectsForOrganization(ctx, org.ID)
```

Comment:
Doing this instead of `select * from projects p join users_project_roles upr on p.project_id = upr.project_id where p.organization_id = ? and upr.user_id = ? limit 1` as I feel this is less expensive and straightforward.

Reply:
I know it's ugly, but I feel that having something like `database.CheckOrgProjectsHasMemberUser(orgID, userID string) (bool, error)` and being sure we have just one DB query is better.

I'm worried about a case where we have many projects in an org, and get held up by many sequential DB roundtrips before a page load. (We might quickly have dozens of example projects in the `rilldata` org.)